src/sentry/api/serializers/models/dashboard.py

```diff
@@ -143,11 +143,26 @@ def get_attrs(self, item_list, user):
         return result
 
     def serialize(self, obj, attrs, user, **kwargs):
+        page_filter_keys = ["environment", "range", "start", "end"]
+        dashboard_filter_keys = ["releases"]
         data = {
             "id": str(obj.id),
             "title": obj.title,
             "dateCreated": obj.date_added,
             "createdBy": serialize(obj.created_by, serializer=UserSerializer()),
             "widgets": attrs["widgets"],
+            "projects": [project.id for project in obj.projects.all()],
         }
+
+        if obj.filters is not None:
+            for key in page_filter_keys:
+                if obj.filters.get(key) is not None:
+                    data[key] = obj.filters[key]
+
+            for key in dashboard_filter_keys:
+                if obj.filters.get(key) is not None:
+                    if data.get("filters") is None:
+                        data["filters"] = {}
+                    data["filters"][key] = obj.filters[key]
```

Comment:
nit: you can probably use [setdefault](https://docs.python.org/3/library/stdtypes.html#dict.setdefault) here to make this a 1 liner
```suggestion
                    data.setdefault("filters", {})[key] = obj.filters[key]
```

Reply:
Avoided this by initializing an empty dict as we discussed for filters.